lib/components/user/monitored-trip/trip-monitored-days.tsx

```diff
@@ -0,0 +1,73 @@
+import { FormattedList, FormattedMessage } from 'react-intl'
+
+import FormattedDayOfWeek from '../../util/formatted-day-of-week'
+import InvisibleA11yLabel from '../../util/invisible-a11y-label'
+import React from 'react'
+import Strong from '../../util/strong-text'
+
+import styled from 'styled-components'
+
+interface Props {
+  days: string[]
+}
+
+const DayCircleContainer = styled.div`
+  display: flex;
+  gap: 4px;
+`
+
+const MonitoredDay = styled.div<{ monitored: boolean }>`
+  width: 27px;
+  height: 27px;
+  border-radius: 50%;
+  background-color: ${(props) => (props.monitored ? '#4152a4' : 'transparent')};
```

Comment:
This is nice! I think a border of the same color should be shown for all cases, so that the outline of the circle is visible for non-monitored days.

Reply:
I prefer the "after" version!

Reply:
Am I missing CSS in the screenshot below?
<img width="289" alt="image" src="https://github.com/opentripplanner/otp-react-redux/assets/56846598/d8d9922e-a8aa-48e0-aec2-e4a42e2a214d">

Reply:
Rats, I thought [beb2dd2](https://github.com/opentripplanner/otp-react-redux/pull/1151/commits/beb2dd287e1167bf49f8cc091c82eea6fe03aa43) fixed this but I guess not. Is it because I'm calling `getBaseColor` at the top level instead of inside the `MonitoredDays` component?

Reply:
Much better!

Reply:
@miles-grant-ibigroup @binh-dam-ibigroup  removed the border in accordance with our tech call conversation - we can use borders on the buttons inside the form, hopefully to show that those are interactive and these aren't?

Reply:
@amy-corson-ibigroup Hmm I think each day should still show the circle shape somehow. Maybe in some pale grey? Contrast-wise, it should be ok, as the high-contrast circle denotes a selected day vs. a low-contrast.